.github/workflows/build-and-publish.yml

```
@@ -0,0 +1,167 @@
+        ls -la ./packages/
+        
+    - name: Create release archive
+      run: |
+        cd release-artifacts
+        zip -r ../bento-sdk-${{ steps.version.outputs.version }}.zip Bento.dll Bento.pdb Bento.deps.json
+        cd ..
+        echo "Created archive: bento-sdk-${{ steps.version.outputs.version }}.zip"
+        ls -la bento-sdk-${{ steps.version.outputs.version }}.zip
+        
+    - name: Create GitHub Release
+      uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)